pkg/daemon/certs/certs.go

```diff
@@ -25,27 +25,103 @@ import (
 	"encoding/pem"
 	"errors"
 	"fmt"
+	"io"
 	"os"
+	"reflect"
 	"syscall"
 	"unsafe"
 
 	"golang.org/x/sys/windows"
 	"k8s.io/klog/v2"
 )
 
-// Reconcile ensures the certificates in the given trusted CA bundle file are imported as system certificates.
+// importedCABundleFile tracks all the certs that the operator has imported into the node's local trust store
+const importedCABundleFile = "C:\\Temp\\imported-certs.pem"
```

Comment:
I can't find any info on this being an actually temp directory.
Any reason to not use `c:\\k`?
I have a feeling we may need to lock our file permissions down more securely, and keeping all WMCO/WICD files contained within c:\k simplifies that.

Reply:
no reason, I can change this to c:\k